apps/journeys-admin/src/components/SignIn/PasswordPage/PasswordPage.tsx

```diff
@@ -36,7 +41,7 @@ export function PasswordPage({
       .required(t('Please enter your email address')),
     password: string().required(t('Enter your password'))
   })
-  async function handleLogin(values, { setFieldError }): Promise<void> {
+  async function handleLogin(values: Values, { setFieldError }): Promise<void> {
```

Comment:
```suggestion
  async function handleLogin(values: values: InferType<typeof validationSchema>, { setFieldError }): Promise<void> {
```

```
import { InferType } from 'yup'
```

This will force the schema to stay in sync.